### DISCUSSION TITLE
Undocumented "published" signal in Wagtail

### DISCUSSION THREAD

**Discussion by NotANormalNerd** [wagtail/wagtail#11460] [Q&A] [answered]
Hello everyone,

I have encountered some undocumented Signals, specifically "published" and "unpublished" in `wagtail/signals.py`.

Current documentation: https://docs.wagtail.org/en/stable/reference/signals.html

I was wondering if the documentation was just forgotten or if this is a intentionally undocumented feature, because it is not considered public API and deemed to be changing in the future.

We are currently working on implementing Snippets and would love to hook into this especially when we schedule Snippets for publication.

Best regards,
Dennis

**Comment by lb-** [comment, marked as answer]
I think it's probably fine to create an issue requesting this.

See also https://github.com/wagtail/wagtail/issues/6991 & https://github.com/wagtail/wagtail/issues/4888
